projects/core/src/model/cart.model.ts

```diff
@@ -41,23 +53,35 @@ export interface Voucher {
   voucherCode?: string;
 }
 
+/**
+ * @deprecated since 5.0, use cart lib instead
+ */
 export interface DeliveryOrderEntryGroup {
   deliveryAddress?: Address;
   entries?: OrderEntry[];
   quantity?: number;
   totalPriceWithTax?: Price;
 }
 
+/**
+ * @deprecated since 5.0, use cart lib instead
+ */
 export interface Principal {
```

Comment:
this one is used by `Order`, so it should go to order-lib, right?

Reply:
If it's _only_ used by order, maybe.  Otherwise Order will depend on cart lib so it can tay in cart lib.

Reply:
`Principal` is imported in `order`, `cart`,  `organization ` ( in order approvals).